bundles/org.openhab.binding.androidtv/pom.xml

```diff
@@ -14,6 +14,10 @@
 
   <name>openHAB Add-ons :: Bundles :: AndroidTV Binding</name>
 
+  <properties>
+    <bnd.importpackage>!org.simpleframework.xml.*,!org.cups4j,!net.sf.ehcache.*,!net.spy.*</bnd.importpackage>
+  </properties>
```

Comment:
I guess they are not needed? cups4j printing library?

Reply:
I pulled that in from the original binding.  Not entirely sure why it was blocked out.  I assume there was a reason at some point.

Reply:
I would suggest to remove it and perform some basic tests. This should not be needed.

Reply:
@wborn could you perhaps help me out here on why this happens? I try to understand this, but fail :-/

Reply:
I was able to remove the first two and not break the compile.  The last two are necessary apparently.

Reply:
Maybe try `!net.sf.ehcache*` instead of `!net.sf.ehcache.*` ?

Reply:
This seems to work without issues...

`<bnd.importpackage>!net.sf.ehcache.*,!net.spy.*</bnd.importpackage>`